<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>451</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar:tab {background-color: rgb(85, 85, 0);color: white;}
QTabBar::tab:selected {background-color: rgb(48, 199, 184); }
QTabBar::tab:hover{background-color: rgb(0, 81, 255);}
QTabBar::tab { height: 30px;width:165px;}
QTabBar::tab:pressed{background-color: rgb(48, 199, 184);}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 127);</string>
     </property>
     <attribute name="title">
      <string>تسجيل الدخول</string>
     </attribute>
     <widget class="QLineEdit" name="lin_user">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: rgb(85, 255, 255);
  border-radius: 8px;
	border:1px solid rgb(85, 0, 127);
	color: rgb(85, 85, 0);
}

QLineEdit:hover{
  border:1.5px solid rgb(85, 85, 0);
}




</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lin_pass">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: rgb(85, 255, 255);
  border-radius: 8px;
	border:1px solid rgb(85, 0, 127);
	color: rgb(85, 85, 0);
}

QLineEdit:hover{
  border:1.5px solid rgb(85, 85, 0);
}




</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enter">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgb(0, 170, 127);
  border-radius: 8px;
  color: #FFFFFF;
	font: 75 14pt &quot;MS Shell Dlg 2&quot;;
	
}
QPushButton:hover{
  background-color: #F082AC;
}
QPushButton:pressed{
background-color: rgb(0, 0, 255);
}




</string>
      </property>
      <property name="text">
       <string>دخول</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>إسم المستخدم</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>كلمة المرور</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>370</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 127);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;إعداد الطالب : علاء عبدالسلام الزيدان&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Create">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 127);</string>
     </property>
     <attribute name="title">
      <string>إنشاء حساب</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>إسم المستخدم</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>الإسم الكامل</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lin_Cname">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: rgb(85, 255, 255);
  border-radius: 8px;
	border:1px solid rgb(85, 0, 127);
	color: rgb(85, 85, 0);
}

QLineEdit:hover{
  border:1.5px solid rgb(85, 85, 0);
}




</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lin_Cuser">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: rgb(85, 255, 255);
  border-radius: 8px;
	border:1px solid rgb(85, 0, 127);
	color: rgb(85, 85, 0);
}

QLineEdit:hover{
  border:1.5px solid rgb(85, 85, 0);
}




</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgb(0, 170, 127);
  border-radius: 8px;
  color: #FFFFFF;
	font: 75 14pt &quot;MS Shell Dlg 2&quot;;
	
}
QPushButton:hover{
  background-color: #F082AC;
}
QPushButton:pressed{
background-color: rgb(0, 0, 255);
}




</string>
      </property>
      <property name="text">
       <string>إنشاء حساب</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lin_CEmile">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: rgb(85, 255, 255);
  border-radius: 8px;
	border:1px solid rgb(85, 0, 127);
	color: rgb(85, 85, 0);
}

QLineEdit:hover{
  border:1.5px solid rgb(85, 85, 0);
}




</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lin_Cpass">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>270</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: rgb(85, 255, 255);
  border-radius: 8px;
	border:1px solid rgb(85, 0, 127);
	color: rgb(85, 85, 0);
}

QLineEdit:hover{
  border:1.5px solid rgb(85, 85, 0);
}




</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>كلمة المرور</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>البريد الإلكتروني</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
